src/misc/folderDetails.ts

```diff
@@ -14,16 +14,18 @@ export class FolderDetails {
     public toZip: string;
     public infoFile: string;
     public mainFile: string;
+    public imageIconFile: string;
     public info: IAppInfo;
 
     constructor(private command: Command) {
         this.setFolder(process.cwd());
         this.mainFile = '';
+        this.imageIconFile = '';
         this.info = {} as IAppInfo;
     }
 
-    public async doesFileExist(file: string): Promise<boolean> {
-        return await fs.pathExists(file) && fs.statSync(file).isFile();
+    public  doesFileExist(file: string): boolean {
+        return  fs.existsSync(file) && fs.statSync(file).isFile();
```

Comment:
Why the switch to sync?

Reply:
I was not aware of the fs-extra npm package and hence thought this function won't work. It's resolved now